Assets/Scripts/Entities/Entity.cs

```diff
@@ -3,34 +3,61 @@
 
 namespace Entities
 {
+    /// <summary>
+    /// An Entity is an instance that can attack, receive attacks, can move and die
```

Comment:
Würde eher sagen: receive attacks → be attacked
Aber ist jetzt eigentlich nicht wirklich relevant :D